core/templates/domain/improvements/ineffective-feedback-loop-task.model.ts

```diff
@@ -90,7 +62,7 @@ export class IneffectiveFeedbackLoopTaskObjectFactory {
     });
   }
 
-  createNew(
+  static createNew(
```

Comment:
don't use "this" inside static functions

Reply:
Done!